lib/serve/rest-api/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Use an argument for the base image
 ARG BASE_IMAGE
-FROM ${BASE_IMAGE}
+FROM --platform=linux/amd64  ${BASE_IMAGE}
```

Comment:
The architecture set in `/lib/core/` is set to `x86`. We need to investigate the correct platform.